src/fides/api/main.py

```diff
@@ -269,6 +272,8 @@ async def setup_server() -> None:
     if not scheduler.running:
         scheduler.start()
 
+    initiate_scheduled_batch_consent_email_send()
```

Comment:
Currently using APScheduler instead of celery beat, as our default mode for running fides is `task_always_eager=True` (without workers), and these scheduled tasks don't seem to work in that mode.